### ISSUE TITLE
Ask for test set

### ISSUE THREAD

**Issue by xxxsy11** [PKU-YuanGroup/ConsisID#12]
Thank you for your excellent work, could you please share your test set in the paper ('we select 30 persons who were not included in the training data and sourced five high-quality images for each ID from the internet.') so I can measure the metrics on my own machine?

**Comment by SHYuanBest** [comment]
Thanks for your interest. We will open source the test set and the code of calculating metrics in the next few days. (under organized)

**Comment by SHYuanBest** [comment]
We have released the [test set](https://huggingface.co/datasets/BestWishYsh/ConsisID-preview-Data/tree/main/eval) and [metric calculation code](https://github.com/PKU-YuanGroup/ConsisID/tree/main/eval), welcome to report.

**Comment by xxxsy11** [comment]
Thank you very much for your work, I've just sent you an email to your email address, please check it if it's convenient!
